src/preprocessing.ipynb

```diff
@@ -37,7 +37,7 @@
     },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        return pd.read_csv(r'C:\Users\pavle\OneDrive\Desktop\TCDSB - Schools - 4326.csv', sep = ',')
</pre><p>Please use relative paths. See my example with satellite.png. The prefix "../" means previous folder.</p><p> </p><p>Also, can you rename the csv file to a more friendly name :)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geotrieu/transit-ai/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>